TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -337,7 +339,8 @@ - (NSArray *)getLinkedLogins {
 }
 
 - (void)unlinkLogin:(TradeItLinkedLogin *)login
-          localOnly:(BOOL)localOnly {
+          localOnly:(BOOL)localOnly
```

Comment:
When the Yahoo apps sync from their backend, you don't want to call `/oAuthDelete` because the user could have signed in with a different Yahoo account and you don't want to unlink all the linked brokers associated with their other account.